model/src/main/java/com/epam/esm/entity/user/UserRole.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.esm.entity.user;
+
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Roles of a user.
+ *
+ * @author Yauheni Tsitou
+ */
+public enum UserRole {
+    USER(Arrays.asList(Permission.USERS_READ, Permission.ORDERS_CREATE,Permission.ORDERS_READ)),
+    ADMIN(Arrays.asList(Permission.values()));
+    private final Set<Permission> permissions;
+
+    UserRole(Collection<Permission> collection) {
+        this.permissions = new HashSet<>(collection);
+    }
+
+    public Set<Permission> getPermissions() {
+        return permissions;
+    }
+
+    public Set<SimpleGrantedAuthority> getAuthorities() {
```

Comment:
This method seems to be out of `UserRole` enum's responsibility.  This enum shouldn't know anything about `SimpleGrantedAuthority`

Reply:
Done.